docs/components/merak_compute.md

```diff
@@ -27,6 +27,7 @@ Based on the requests, it will invoke workers via the temporal client to run the
 
 ### VM Workers
 The VM Worker will be responsible for making calls to the Merak Agent to Create/Update/Delete VMs by running workflows.
+
```

Comment:
@phudtran Can you describe more details about VM workers here, for example, each merak-agent has dedicated 1 VM workers, does VM workers can be shared by multiple merak-agents, and how grpc channel got established between merak-compute and merak-agent througth VM workers or workflow?

Reply:
Sure. I've updated the VMCreate workflow section with this information.